### ISSUE TITLE
Domain Hijacking: Removal of "henryzhu.me" CNAME 

### ISSUE THREAD

**Issue by hzhu** [henryee333/henryzhu.github.io#1]
**TLDR: I have not permitted to use `"henryzhu.me"` in any repositories besides my own. The creation of [this CNAME file](https://github.com/henryee333/henryzhu.github.io/blob/master/CNAME#L1) hijacks my domain. Please remove the CNAME file (or modify it to your domain `"henryzhu.xyz"`). I would greatly appreciate the removal of my domain.**

----------------------------

Greetings @henryee333,
 
Last month I noticed that my website http://henryzhu.me was not working:

<img src="https://user-images.githubusercontent.com/1811365/69920307-38fdbb00-143b-11ea-9c30-1bcfb6e7c036.png" width="400">

It is supposed to redirect to http://hzhu.github.io:

<img src="https://user-images.githubusercontent.com/1811365/69920318-621e4b80-143b-11ea-84c9-c62c7072bf52.png" width="400">

Then I realized my CNAME was taken:

<img alt="Screen Shot 2019-11-16 at 3 39 40 AM" src="https://user-images.githubusercontent.com/1811365/69920580-a232fd80-143e-11ea-8ae7-20aea6bb0c7b.png" width="400">

This repository does not back a working GitHub page and contains the CNAME file with my domain, `henryzhu.me`. It _seems_ that this repository is used for GitHub domain hijacking as described in @jehy's Medium article, [_**Hijacking domain using Github pages**_](https://medium.com/@jehy/hijacking-domain-using-github-pages-41c80ac57523). Like @jehy, I searched for my domain on GitHub:

<img alt="Screen Shot 2019-11-16 at 4 00 50 AM" src="https://user-images.githubusercontent.com/1811365/69920572-90515a80-143e-11ea-8370-9267e1a0916e.png" width="400">

I noticed that you worked on **theDesignExchange**:

<img width="1044" alt="Screen Shot 2019-11-16 at 4 23 18 AM" src="https://user-images.githubusercontent.com/1811365/71315403-5823b300-2411-11ea-920d-a93f2aebd862.png">

I noticed that a UC Berkeley email was used:

<img width="746" alt="Screen Shot 2019-11-16 at 4 25 04 AM" src="https://user-images.githubusercontent.com/1811365/71315420-bb154a00-2411-11ea-9e74-faf0b31efd94.png">

So I did a Google and LinkedIn search and found that the username is associated with a computer science student at UC Berkeley who also worked on **theDesignExchange**.

<img width="1680" alt="Screen Shot 2019-11-16 at 3 14 11 AM" src="https://user-images.githubusercontent.com/1811365/71315425-e4ce7100-2411-11ea-828f-6652f852f994.png">

<img width="1680" alt="Screen Shot 2019-11-16 at 3 15 20 AM" src="https://user-images.githubusercontent.com/1811365/71315436-38d95580-2412-11ea-82ea-a2198bea3bc0.png">

I have not permitted to use `"henryzhu.me"` in any repositories besides my own. The creation of [this CNAME file](https://github.com/henryee333/henryzhu.github.io/blob/master/CNAME#L1) hijacks my domain. Please remove the CNAME file (or modify it to your domain `"henryzhu.xyz"`). I would greatly appreciate the removal of my domain.
